### DISCUSSION TITLE
SPHinXsys on HPC clusters

### DISCUSSION THREAD

**Discussion by hasantahir** [Xiangyu-Hu/SPHinXsys#216] [Q&A] [answered]
Having some examples of using SPHinXsys on HPC clusters would be great to have somewhere in the documentation.

Many thanks!

**Comment by Xiangyu-Hu** [comment, marked as answer]
Sorry for the delayed reply.
Actually, the SPHInXsys currently is targeting single-node computation.
So MPI, which is for multi-node HPC cluster, has not been implemented.
Xiangyu

**Reply by hasantahir** [reply]
Thanks for getting back. I'm really looking forward to future updates and it's great to see how much work has gone into developing SPHInXys. Keep up the good work!
